server/src/controllers/Events/resolver.ts

```diff
@@ -984,35 +973,10 @@ ${unsubscribeOptions}`,
       event.chapter.user_bans.map(({ user_id }) => user_id),
     );
 
-    const users: User[] = [];
-    if (emailGroups.includes('interested')) {
-      const interestedUsers =
-        event.chapter.chapter_users?.filter(
-          ({ subscribed, user_id }) =>
-            !bannedUserIds.has(user_id) && subscribed,
-        ) ?? [];
-
-      users.push(...interestedUsers);
-    }
-
-    if (emailGroups.includes('on_waitlist')) {
-      const waitlistUsers = event.event_users.filter(
-        ({ rsvp }) => rsvp.name === 'waitlist',
-      );
-      users.push(...waitlistUsers);
-    }
-    if (emailGroups.includes('confirmed')) {
-      const confirmedUsers = event.event_users.filter(
-        ({ rsvp }) => rsvp.name === 'yes',
-      );
-      users.push(...confirmedUsers);
-    }
-    if (emailGroups.includes('canceled')) {
-      const canceledUsers = event.event_users.filter(
-        ({ rsvp }) => rsvp.name === 'no',
-      );
-      users.push(...canceledUsers);
-    }
+    const users: User[] =
```

Comment:
```suggestion
    const users =
```
Explicit type is no longer needed, along the interface definition (not possible to suggest in github).